zipkin-web/src/main/scala/com/twitter/zipkin/web/TraceSummary.scala

```diff
@@ -18,7 +18,6 @@ package com.twitter.zipkin.web
 
 import com.twitter.finagle.tracing.SpanId
 import com.twitter.zipkin.common.{Trace, Endpoint, Span, SpanTreeEntry}
-import com.twitter.zipkin.web.Util.getRootMostSpan
 
 case class SpanTimestamp(name: String, timestamp: Long, duration: Long) {
```

Comment:
Agreed.. Fwiw if we accept gzip (in the javascript client) and the limit is
10, probably have some time to sort out what's best.